.github/workflows/android-ci.yml

```diff
@@ -35,3 +35,6 @@ jobs:
       run: |
         tools/generate-files
         tools/android-test
+    - name: Build sample app
+      run: |
```

Comment:
make sure you reuse lib for iOS and Android, you can use `needs` option on the job to wait until that's done

Reply:
Good hint, but it looks to me that steps are executed in order.  There is no explicit dependency specified in the coverage collection, wouldn't work if is not in order.